app/assets/stylesheets/application.css.scss

```diff
@@ -30,12 +30,9 @@ body {
   @media screen and (max-width: 39.9375em) {
     width: 100%;
   }
-  @media screen and (min-width: 40em) and (max-width: 63.9375em) {
+  @media screen and (min-width: 40em) {
     width: 60%;
```

Comment:
Instead of declaring % of width in css you could use foundations rows / columns to specify how many columns you want the container to have at each breakpoint.

Also for bonus points you could setup custom breakpoints e.g the following which you can put in your foundation_and_overrides.scss file ( though you'd need gem 'foundation-rails' in your Gemfile followed by rails g foundation:install to install foundation in your project )


$small-range: (0em, 40em);
$medium-range: (40.063em, 63.9312em);
$large-range: (63.9375em, 90em);
$xlarge-range: (90.063em, 160em);
$xxlarge-range: (160.063em, 99999999em);

then something like 

$small-up: $screen;
$small-only: "#{$screen} and (max-width: #{upper-bound($small-range)})";

$medium-up: "#{$screen} and (min-width:#{lower-bound($medium-range)})";
$medium-down: "#{$screen} and (min-width:#{lower-bound($small-range)}) and (max-width:#{upper-bound($medium-range)})";
$medium-only: "#{$screen} and (min-width:#{lower-bound($medium-range)}) and (max-width:#{upper-bound($medium-range)})";

$med-large-up: "#{$screen} and (min-width:#{$med-large})";
$med-large-down: "#{$screen} and (max-width:#{$med-large})";

$large-up: "#{$screen} and (min-width:#{lower-bound($large-range)})";
$large-down: "#{$screen} and (min-width:#{lower-bound($small-range)}) and (max-width:#{upper-bound($large-range)})";
$large-only: "#{$screen} and (min-width:#{lower-bound($large-range)}) and (max-width:#{upper-bound($large-range)})";

$xlarge-up: "#{$screen} and (min-width:#{lower-bound($xlarge-range)})";
$xlarge-down: "#{$screen} and (min-width:#{lower-bound($small-range)}) and (max-width:#{upper-bound($xlarge-range)})";
$xlarge-only: "#{$screen} and (min-width:#{lower-bound($xlarge-range)}) and (max-width:#{upper-bound($xlarge-range)})";

$xxlarge-up: "#{$screen} and (min-width:#{lower-bound($xxlarge-range)})";
$xxlarge-only: "#{$screen} and (min-width:#{lower-bound($xxlarge-range)}) and (max-width:#{upper-bound($xxlarge-range)})";

Reply:
Then you can do things like
 
@media #{$large-up} {

}